model/note/markdown.go

```diff
@@ -36,11 +36,30 @@ func markdownSerializer(images []*Image) *markdown.Serializer {
 		"bulletList":  vanilla.Nodes["bullet_list"],
 		"orderedList": vanilla.Nodes["ordered_list"],
 		"listItem":    vanilla.Nodes["list_item"],
-		"heading":     vanilla.Nodes["heading"],
-		"blockquote":  vanilla.Nodes["blockquote"],
-		"rule":        vanilla.Nodes["horizontal_rule"],
-		"hardBreak":   vanilla.Nodes["hard_break"],
-		"image":       vanilla.Nodes["image"],
+		"decisionList": func(state *markdown.SerializerState, node, _parent *model.Node, _index int) {
+			if node.Attrs == nil {
+				node.Attrs = map[string]interface{}{}
+			}
+			node.Attrs["tight"] = true
+			state.RenderList(node, "  ", func(_ int) string { return "✍ " })
+		},
+		"decisionItem": vanilla.Nodes["list_item"],
+		"taskList": func(state *markdown.SerializerState, node, _parent *model.Node, _index int) {
+			state.RenderList(node, "  ", func(_ int) string { return "- " })
+		},
+		"taskItem": func(state *markdown.SerializerState, node, _parent *model.Node, _index int) {
+			if node.Attrs["state"] == "DONE" {
+				state.Write("[X] ")
+			} else {
+				state.Write("[ ] ")
+			}
+			state.RenderContent(node)
```

Comment:
It was for that!! I forgot to call `RenderContent` when I was trying to do that: 
```
"taskItem": func(state *markdown.SerializerState, node, _parent *model.Node, _index int) {
			var cross = " "
			value, _ := node.Attrs["state"].(string)
			if (value) == "DONE" {
				cross = "X"
			}
			state.Write(fmt.Sprintf("[%s]", cross))
			//state.Text(node.TextContent(), false)
		},
```

Is there a simple rule to know if we should call `RenderContent()` or `Text()`? 

`RenderInline` or `RenderList` seems to speak for themselves

Reply:
If the thing that you want to render can have links or bold text, you should call `RenderContent()`, but if it is expected to be a raw text with no decoration (like in a block of code), you should call `Text()`.